package.json

```diff
@@ -28,7 +31,7 @@
     "mocha": "~5.0.4"
   },
   "engines": {
-    "node": ">=4.5"
+    "node": ">=5.10"
```

Comment:
What motivates this change?

Reply:
Stated in the PR description: `Updated min node version from v4 to v5.10 to use Buffer.from instead of the deprecated new Buffer api`

Reply:
It wasn't necessary to change this as part of this PR, if you prefer more piecemeal changes, happy to split it up. But I don't think think new `buffer-layout` releases need this far back of node support. Folks using v4 can use old versions of `buffer-layout`

Reply:
OK.  Please put that sort of information into the commit messages or code comments.  Maintainers (me) should not have to go back to github to look at the PR to figure out what's going on.

Reply:
Sounds great, thanks for the feedback!